my-app/src/components/RecurringBills/RecurringBills.tsx

```diff
@@ -1,5 +1,250 @@
-import React from "react";
+import React, { FC, useState, useMemo } from "react";
+import {
+  StyledWrapper,
+  SectionHeader,
+  StyledWrapperDetails,
+  StyledWrapperDetailsHeader,
+  TotalBills,
+  SummaryBills,
+  StyledSummaryBillsItem,
+  BillsWrapper,
+  WrapperImage,
+  IconBill,
+} from "./RecurringBills.styles";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@mui/material";
+import { useSelector } from "react-redux";
+import { RootState } from "../../store/store";
+import {
+  getRecurringBills,
+  getSummaryRecurringBillsData,
+  sortMethods,
+} from "../../utils/transactionsUtils.ts";
+import { SelectDropdown } from "../SelectDropdown/SelectDropdown.tsx";
+import { InputSearch } from "../InputSearch/InputSearch.tsx";
 
-export const RecurringBills = () => {
-  return <></>;
+type SortOption =
+  | "All"
+  | "Highest"
+  | "Lowest"
+  | "Oldest"
+  | "Latest"
+  | "A to Z"
+  | "Z to A";
+
+const tableHeaders = ["Bill Title", "Due Date", "Amount"];
+
+const sortByDayOfMonth = (transactions: TransactionsProps[]) =>
+  [...transactions].sort((a, b) => {
```

Comment:
if `transactions` is an `array`, then:
`transactions` is the same as `[...transactions]` and the same as `[...[...transactions]]`, etc etc.

Hence:
```suggestion
  transactions.sort((a, b) => {
```

Reply:
@apawlik96 just fix this comment, and this PR is ✅.
Once you fix this, feel free to merge without waiting for further approval.